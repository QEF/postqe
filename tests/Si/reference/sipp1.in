 &inputpp
   plot_num=1,
   prefix='Si',
   outdir='./tmp/'
   filplot='ppout1',
/
