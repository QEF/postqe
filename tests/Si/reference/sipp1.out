
     Program POST-PROC v.6.1 starts on 17Jul2017 at 11:16:15 

     This program is part of the open-source Quantum ESPRESSO suite
     for quantum simulation of materials; please cite
         "P. Giannozzi et al., J. Phys.:Condens. Matter 21 395502 (2009);
          URL http://www.quantum-espresso.org", 
     in publications or presentations arising from this work. More details at
     http://www.quantum-espresso.org/quote

     Parallel version (MPI), running on     4 processors
     R & G space division:  proc/nbgrp/npool/nimage =       4

   Info: using nr1, nr2, nr3 values from input

   Info: using nr1, nr2, nr3 values from input

     IMPORTANT: XC functional enforced from input :
     Exchange-correlation      = PZ ( 1  1  0  0 0 0)
     Any further DFT definition will be discarded
     Please, verify this is what you really want


     Parallelization info
     --------------------
     sticks:   dense  smooth     PW     G-vecs:    dense   smooth      PW
     Min          87      87     30                 1068     1068     216
     Max          88      88     31                 1071     1071     218
     Sum         349     349    121                 4279     4279     869


     Calling punch_plot, plot_num =   1
     Writing data to file  ppout1
     Message from routine chdens:
     namelist plot not found or invalid, exiting

     POST-PROC    :     0.18s CPU         0.21s WALL


   This run was terminated on:  11:16:15  17Jul2017            

=------------------------------------------------------------------------------=
   JOB DONE.
=------------------------------------------------------------------------------=
